Source/Particles/MultiParticleContainer.H

```diff
@@ -211,7 +211,10 @@ public:
 
     void WriteHeader (std::ostream& os) const;
 
-    void SortParticlesByBin (amrex::IntVect bin_size);
+    void SortParticlesByBin (
+        const amrex::IntVect& bin_size,
+        bool sort_particles_for_deposition,
```

Comment:
Should this be `const` like in the .cpp file, or is that something that clang-tidy does not like for some reason (I can't remember)?
```suggestion
        const bool sort_particles_for_deposition,
```

Reply:
You can put `const` in the header file as well (the code will compile and I am almost sure that `clang-tidy` won't complain). However, from the point of view of someone calling the function, it doesn't change anything, since the variable is passed by value. The fact that you don't change the value of `sort_particles_for_deposition` inside the function can be considered an implementation detail. That's why I put `const` only in the cpp file.  We can, of course, decide to use a different convention! There's some debate on this topic on stackoverflow, e.g. [here](https://stackoverflow.com/questions/1554750/is-const-necessary-in-function-parameters-when-passing-by-value)

Reply:
Okay, that's fine by me. I understand that it does not make any difference, but I thought it could potentially increase "transparency", especially if it doesn't have negative side effects. In some sense, I was asking myself "why not?" instead of "why?". But I'm okay with leaving it only in the implementation.